src/util/constants.ts

```diff
@@ -7,3 +7,5 @@ export const NOT_IMPLEMENTED_TEXT =
   "This feature is not implemented yet. This will allow you to compile Oxido code to WebAssembly in the future." as const
 
 export const VERSIONS_API = "https://api.github.com/repos/oxidic/oxido/releases" as const
+
+export const README_URL = "https://raw.githubusercontent.com/oxidite/oxido/main/README.md" as const
```

Comment:
```suggestion
export const README_URL = "https://raw.githubusercontent.com/oxidic/oxido/main/README.md" as const
```
oxidite is no longer the org name